package.json

```diff
@@ -1,48 +1,39 @@
 {
+    "private": true,
+    "workspaces": ["packages/*"],
     "name": "@stackoverflow/stacks",
-    "description": "Stack Overflow’s CSS and Design Pattern Library. Stacks is an atomic CSS library with classes and components for rapidly building Stack Overflow.",
     "repository": {
         "type": "git",
         "url": "https://github.com/StackExchange/Stacks.git"
     },
-    "version": "2.8.4",
-    "files": [
-        "dist",
-        "lib",
-        "!lib/**/*.test.*"
-    ],
-    "main": "./dist/js/stacks.js",
-    "types": "./dist/index.d.ts",
-    "style": "./dist/css/stacks.css",
-    "less": "./lib/stacks.less",
-    "unpkg": "dist/css/stacks.min.css",
+    "license": "MIT",
     "scripts": {
         "start": "concurrently -n w: npm:start:*",
         "build": "concurrently -n w: npm:build:*",
-        "build:bundle": "webpack --mode=production",
-        "build:docs": "webpack --mode=production --config ./docs/webpack.config.js && cd ./docs && eleventy",
-        "start:webpack": "webpack --watch --config ./docs/webpack.config.js",
-        "start:eleventy": "cd ./docs && eleventy --serve",
+        "build:bundle": "webpack --mode=production --config ./packages/stacks-classic/webpack.config.js",
+        "build:docs": "webpack --mode=production --config ./packages/stacks-docs/webpack.config.js && cd ./packages/stacks-docs && eleventy",
+        "start:webpack": "webpack --watch --config ./packages/stacks-docs/webpack.config.js",
+        "start:eleventy": "cd ./packages/stacks-docs && eleventy --serve",
         "test": "npm run test:less && npm run test:unit && npm run test:a11y && npm run test:visual",
-        "test:a11y": "web-test-runner --group=a11y",
-        "test:unit": "web-test-runner --group=unit",
-        "test:unit:watch": "web-test-runner --group=unit --watch",
+        "test:a11y:ci": "cd ./packages/stacks-classic && web-test-runner --group=a11y",
```

Comment:
We should avoid `cd`ing into folder and instead use the npm workspace functionalities. I have added a comment to this file that should help simplify all of this. 
I can imagine something like this here
```
"test:a11y": "npm run test:a11y --workspaces --if-present"
```
and keep the original script as it was in the `packages/stacks-classic/package.json`

The same goes with pretty much for all the other scripts: formatting, linting, etc...